javascript/app.js

```diff
@@ -107,6 +123,42 @@ FinancialFreedom.directive("btnOutputControl", function($filter){
     }
 });
 
+FinancialFreedom.controller('AboutController', ['$scope', function($scope) {
+    $scope.templates =
+    [ { name: 'Privacy Policy', url: 'partials/legal/privacy_policy.html'},
+      { name: 'Terms of Service', url: 'partials/legal/terms_of_service.html'} ];
+    
+    $scope.template = null;
+    $scope.privacy_is_active = false;
+    $scope.privacy_is_toc = false;
+
+    $scope.revealLegalDoc = function(doc) {
+        if (doc == 'privacy-policy') {
+            
+            if ($scope.privacy_is_active == true) {
+                $scope.template = null;
+                $scope.privacy_is_active = false;
+            } else {
+                $scope.template = $scope.templates[0];
+                $scope.privacy_is_active = true;
+                $scope.toc_is_active = false;
+            }
+        }
+        else if (doc == 'toc') {
+
+            if ($scope.toc_is_active == true) {
+                $scope.template = null;
+                $scope.toc_is_active = false;
+            } else {
+                $scope.template = $scope.templates[1];
+                $scope.toc_is_active = true;
+                $scope.privacy_is_active = false;

```

Comment:
could break all these if else blocks into helper functions. So rather than having to explicitly call `$scope.template = null` and such, could just call `hideToc()` which would make it much easier to understand what's going on.